deploy/kubernetes/che-devfile-registry/templates/deployment.yaml

```diff
@@ -56,3 +56,6 @@ spec:
             memory: {{ .Values.cheDevfileRegistryMemoryLimit }}
           requests:
             memory: 256Mi
+        envFrom:
+          - configMapRef:
+              name: che-devfile-registry
```

Comment:
Seems to me that we should make this config map ref `optional`, to avoid update problems.

Reply:
Done.